.lintrunner.toml

```diff
@@ -54,6 +54,10 @@ include_patterns = [
     'torch/csrc/**/*.cpp',
     'test/cpp/**/*.h',
     'test/cpp/**/*.cpp',
+    'third_party/nvfuser/csrc/**/*.h',
+    'third_party/nvfuser/csrc/**/*.cpp',
+    'third_party/nvfuser/test/**/*.h',
+    'third_party/nvfuser/test/**/*.cpp',
```

Comment:
Why not the benchmark files?

Reply:
good question. they were not applied to benchmark files originally... But would be a nice addition~~